js/index.js

```diff
@@ -474,6 +476,10 @@ function searchFeatureColumn() {
     });
     filteredData.toArray().forEach(f => {
       let exp = featureFilter(f);
+
+      // Since future-0-0 does not exist, mark and do not exist by default
+      exp = exp.replaceAll("future_0_0", "=");
```

Comment:
Hi @siva19susi 
why this feaure flag is replaced to `=` char? What is the meaning of `=`?